item.rb

```diff
@@ -20,4 +20,14 @@ def source(source)
   def label(label)
     @label = label
   end
+
+  def can_be_archived?
+    @publish_date > 10
```

Comment:
Good job checking if the `publish_date > 10`. But I kindly suggest validating the `publish_date` that it is `not empty or nil` because we cannot perform computation on an `empty or nil` value.